proxy/Cargo.toml

```diff
@@ -36,8 +35,7 @@ tokio-signal = "0.1"
 prost = "0.3.0"
 prost-types = "0.3.0"
 
-abstract-ns = "0.4"
-ns-dns-tokio = "0.4"
+trust-dns-resolver = { version = "0.8.2", default-features = false, git = "https://github.com/bluejekyll/trust-dns", branch = "0.8_release" }
```

Comment:
@bluejekyll will release trust-dns-resolver 0.8.2 off this branch soon. This branch was created, in part, so we wouldn't need to deal with any hassles associated with the new tokio, which trust-dns-resolver 0.9 uses.

Reply:
fyi: https://www.crates.io/crates/trust-dns-resolver